nuxeo-jsf/nuxeo-platform-contentview-jsf/src/test/java/org/nuxeo/ecm/platform/contentview/jsf/test/TestContentViewState.java

```diff
@@ -307,6 +307,27 @@ public void testSaveJSONContentView() throws Exception {
         assertEquals(ENC_CURRENT_DOC_CHILDREN, encodedJson);
     }
 
+    @Test
+    public void testRestoreJSONContentViewWithNullArray() throws Exception {
+        assertNull(service.saveContentView(null));
+
+        String json = "{" + "\"contentViewName\":\"SEARCH_DOCUMENT\"," + "\"pageSize\":2,"
+                + "\"currentPage\":0,"
```

Comment:
For readability and resistance to reformatting we usually write this kind of thing with comment markers at the end of each line, like:

        String json = "{" //
                + "\"contentViewName\":\"SEARCH_DOCUMENT\"," //
                + "\"pageSize\":2," //
                + "\"currentPage\":0," //
                ...

Reply:
Good to know, I just copied an example in the exisiting class.